src/main/java/com/denizenscript/depenizen/bukkit/objects/breweryx/BreweryRecipeTag.java

```diff
@@ -0,0 +1,349 @@
+        // @returns ElementTag(Number)
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns the amount of alcohol in a perfect potion.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "alcohol", (attribute, object) -> {
+            return new ElementTag(object.bRecipe.getAlcohol());
+        });
+
+        // <--[tag]
+        // @attribute <BRecipeTag.lore>
+        // @returns ListTag(ElementTag)
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns a ListTag of the lore of the recipe (displayed on potion).
+        // -->
+        tagProcessor.registerTag(ListTag.class, "lore", (attribute, object) -> {
+            if (object.bRecipe.getLore() == null) {
+                return null;
+            }
+            ListTag lore = new ListTag();
+            for (Tuple<Integer, String> tuple : object.bRecipe.getLore()) {
+                lore.addObject(new ElementTag(tuple.second()));
+            }
+            return lore;
+        });
+
+        // <--[tag]
+        // @attribute <BRecipeTag.custom_model_data>
+        // @returns ListTag(ElementTag(Number))
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns a ListTag of the 3 possible custom model data's for each varied quality of the recipe/potion.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "custom_model_data", (attribute, object) -> {
+            ListTag cmDatas = new ListTag();
+            for (int cmData : object.bRecipe.getCmData()) {
+                cmDatas.addObject(new ElementTag(cmData));
+            }
+            return cmDatas;
+        });
+
+        // <--[tag]
+        // @attribute <BRecipeTag.effects>
+        // @returns ListTag(ElementTag)
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns a ListTag of potion effects of as their names (Example: SLOW_FALLING).
+        // -->
+        tagProcessor.registerTag(ListTag.class, "effects", (attribute, object) -> {
+            ListTag effects = new ListTag();
+            for (BEffect bEffect : object.bRecipe.getEffects()) {
+                effects.addObject(new ElementTag(bEffect.getType().toString()));
+            }
+            return effects;
+        });
+
+        // <--[tag]
+        // @attribute <BRecipeTag.player_commands>
+        // @returns ListTag(ElementTag)
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns a ListTag of commands that are run by the player when the potion is drunk.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "player_commands", (attribute, object) -> {
+            if (object.bRecipe.getPlayercmds() == null) {
+                return null;
+            }
+            ListTag cmds = new ListTag();
+            for (Tuple<Integer, String> tuple : object.bRecipe.getPlayercmds()) {
+                cmds.addObject(new ElementTag(tuple.second()));
```

Comment:
When constructing an ElementTag just from general plain text mark it explicitly as plaintext with `, true`